kratos/python/add_strategies_to_python.cpp

```diff
@@ -56,6 +56,8 @@
 
 // Builder And Solver
 #include "solving_strategies/builder_and_solvers/builder_and_solver.h"
+#include "solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h"
```

Comment:
this is being discussed in a different PR right? if you only had changes in the ContactStructuralMechanicsApplication it would be much easier to accept the PR

Reply:
#3249 must be approved and merged first